src/main/webapp/js/webrtc_adaptor.js

```diff
@@ -493,15 +492,15 @@ export class WebRTCAdaptor
 		this.webSocketAdaptor.send(JSON.stringify(jsCmd));
 	}
 
-	joinRoom(roomName, streamId) 
+	joinRoom(roomName, streamId, mode) 
```

Comment:
In conference.html sample, I think this method is used with 2 parameters. Does it cause a problem?

Reply:
Mode "legacy" is added for conference.html